src/main/groovy/com/github/prokod/gradle/crossbuild/ResolutionStrategyConfigurer.groovy

```diff
@@ -308,15 +308,19 @@ class ResolutionStrategyConfigurer {
 
         def correctDependencies = dependencyMap[requested.name]
 
-        assert correctDependencies.size() == 1 : 'There should be one candidate to replace offending dependency ' +
+        assert correctDependencies.size() <= 1 : 'There should be one candidate to replace offending dependency ' +
```

Comment:
Here you leverage actually a correction method that was intended for user side typos and bailing out of situations like cross compiling where dependency tree contains a specific version of scala-reflect for instance. I am not sure this is robust enough, I will give it some thought.

Can you provide a test use case ?
Thx